packages/gestalt/src/TextField.jsdom.test.js

```diff
@@ -3,6 +3,8 @@ import { createRef } from 'react';
 import { fireEvent, render } from '@testing-library/react';
 import TextField from './TextField.js';
 
+jest.mock('./contexts/I18nProvider.js');
```

Comment:
I explored more global ways of applying this mock but came up short. We can revisit in the future, hopefully with more luck